docs/source/reservations/reservations.rst

```diff
@@ -0,0 +1,182 @@
+
+
+To request capacity reservations/blocks, see the official docs:
+
+* `AWS Capacity Reservations <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-capacity-reservations.html>`_
+* `AWS Capacity Block Offers <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-capacity-blocks.html>`_
+
+Once you have got a reservation/block, you will get an id of your reservation, like ``cr-0123456789abcdefg``.
+
+You can specify two fields in ``~/.sky/config.yaml``:
+
+* ``aws.prioritize_reservations``: whether to prioritize launching clusters from capacity reservations in any region/zone over on-demand/spot clusters. This is useful to fully utilize your reserved capacity created with *Instance eligibility: open*.
+* ``aws.specific_reservations``: a list of reservation ids that can be used by SkyPilot. This is useful if you have multiple capacity blocks or capacity reservations with *Instance eligibility: targeted* for different instance types in multiple regions/zones.
+
+.. note::
+
+    If any of the fields are specified, please allow about 30 seconds for SkyPilot optimizer to retrieve the latest reservation/block status on all regions and zones from your AWS account.
+
+.. code-block:: yaml
+
+    aws:
+      prioritize_reservations: true
+      specific_reservations:
+        # 1x H100 capacity block in us-east-1
+        - "cr-0123456789"
+        # 2x A100 reservation in us-west-2
+        - "cr-123456789a"
+        # 2x M5a.16xlarge reservation in us-east-1
+        - "cr-23456789ab"
+
+For more details of the configuration, see reference for :ref:`config-yaml`.
+
+
+
+.. _utilizing-reservations:
+
+Utilizing Reservations
+~~~~~~~~~~~~~~~~~~~~~~
+
+By specifying configurations above, SkyPilot will automatically figure out the best place to launch your clusters with available capacity in your reservations/blocks considered as zero-cost.
+
+For example, if you are launching a cluster with the following SkyPilot YAML:
+
+.. code-block:: yaml
+
+    resources:
+      cloud: aws
+      accelerators: A100:8
+    
+    num_nodes: 4
+
+
+SkyPilot will utilize the capacity reservation as follows:
+1. Find available capacity in your capacity block in ``us-west-2`` in the reservation ``cr-123456789a``. Let's say 1 A100 instance capacity is available.
+2. SkyPilot will launch 4 nodes in ``us-west-2`` with 1 node coming from the reservation, and the rest 3 nodes from on-demand.
+3. If it fails to find available A100 instances from on-demand, it will automatically :ref:`failover <auto-failover>` to other regions/zones.
+
+
+.. hint::
+
+    If you have a capacity block with a starting time in the future, you can use SkyPilot to launch a job on that capacity block as soon as the starting date comes, i.e. ``sky jobs launch --region us-east-1 --gpus H100:8 task.yaml``, so you don't have to wake up at 4:30am PDT to launch your job on a newly available capacity block.
```

Comment:
```suggestion
    If you have a capacity block with a starting time in the future, you can use SkyPilot to launch a job on that capacity block at any time using ``sky jobs launch --region us-east-1 --gpus H100:8 task.yaml``. SkyPilot will automatically keep retrying to launch the job, and it will succeed as soon as the capacity block's start time hits. Namely, you don't have to wake up at 4:30am PDT to launch your job on a newly available capacity block.
```